src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(num) {
+      result += num;
+
+      return this;
+    },
+    subtract(num) {
+      result -= num;
+
+      return this;
+    },
+    multiply(num) {
+      result *= num;
+
+      return this;
+    },
+    divide(num) {
+      if (num !== 0) {
+        result /= num;
+
+        return this;
+      } else {
+        throw new Error('Cannot divide by zero!');
+      }
+    },
```

Comment:
The division-by-zero check could be improved to avoid unnecessary nesting.

```suggestion
    divide(num) {
      if (num === 0) {
        throw new Error('Cannot divide by zero!');
      }
      result /= num;
      return this;
    },
```